pkg/util/unstructured.go

```diff
@@ -31,6 +31,7 @@ import (
 
 const (
 	CRDKind                              = "CustomResourceDefinition"
+	CRDResource                          = "customresourcedefinitions"
 	CRDGroup                             = "apiextensions.k8s.io"
```

Comment:
FYI: we do not need our own copy of this string, it is available in https://github.com/kubernetes/apiextensions-apiserver/blob/v0.28.6/pkg/apis/apiextensions/register.go#L24